java/src/main/java/gov/doc/ntia/sigmf/MetaDoc.java

```diff
@@ -101,19 +93,19 @@ public void addCapture(Capture capture) {
   }
 
   /**
-   * Creates an Acquisition (MetaDoc + Data) by reading the data file in
-   * Global dataFilePath.
+   * Creates an Acquisition (MetaDoc + Data) by reading the data file in Global dataFilePath.
+   *
    * @return the Acquisition.
    * @throws IOException when unable to read the file specified in the Global dataFilePath.
    */
   @JsonIgnore
   public Acquisition getAcquisition() throws IOException {
     Acquisition acquisition = new Acquisition();
     acquisition.setMetaDoc(this);
-    log.debug("reading datafile:" + global.getDataFilePath());
     BufferedInputStream inputStream =
-        new BufferedInputStream(new FileInputStream(global.getDataFilePath()));
+        new BufferedInputStream(new FileInputStream(global.getFilepath() + ".sigmf-data"));
     byte[] sensedData = IOUtils.toByteArray(inputStream);
+    inputStream.close();
```

Comment:
Recommend to use try{} finally {} to make sure inputStream gets closed.

Reply:
Refactored to use try with resources